docs/assets.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 5
+sidebar_label: Asset Exchange
+title: Asset Exchange
+description: Exchange your treasury's tokens.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+You can exchange your treasury's tokens to other supported assets and tokens, based on available swap providers.
+
+:::tip
+Near Treasury uses [RHEA Finance](concepts/exchange.md) for all asset exchange features. Learn more about it in [this article](concepts/exchange.md).
+:::
+
+<div class="screenshot">
+
+![Asset Exchange](/img/screens/asset-ex.png)
+
+</div>
+
+**Available actions:**
+
+- [Creating Asset Exchange Requests](#creating-asset-exchange-requests)
+- [Approving or Rejecting Exchange Requests](#approving-or-rejecting-exchange-requests)
+
+---
+
+## Creating Asset Exchange Requests
+
+To exchange one type of cryptocurrency for another:
+
+1.  Go to the `Asset Exchange` section.
+    
+2.  Select the tokens you want to exchange.
+    
+3.  Input the desired amounts.
+    
+4.  Submit the exchange request for approval.
+
+:::info Price slippage
+Exchange rates may fluctuate during the approval process for your asset exchange request.
```

Comment:
can we also please explain a bit
- Minimum received
- pool fee
- Additional Storage Purchase
- Price Deference
<img width="585" alt="image" src="https://github.com/user-attachments/assets/c172e113-c64f-4757-bb4d-07953e5d43ef" />